src/v/cluster/tests/partition_allocator_fixture.h

```diff
@@ -83,6 +93,7 @@ struct partition_allocator_fixture {
         return req;
     }
 
+    ss::sharded<cluster::members_table> members;
```

Comment:
This is quorum state, right? We should consider keeping it separated and adding conveniences for restarting operations when it changes, etc.. Not in this patch, but I think we should discuss this soon. :smile: